Sources/ViewInspector/SwiftUI/CustomView.swift

```diff
@@ -55,14 +55,16 @@ internal extension Content {
     
     func extractCustomView() throws -> Content {
         let inspectable = try Inspector.cast(value: view, type: Inspectable.self)
```

Comment:
It's possible to remove this line now and not cast to `Inspectable`. In this PR we leave the requirement that views conform to `Inspectable` to be inspected.